frontend/src/main.ts

```diff
@@ -1,6 +1,64 @@
 import { createApp } from 'vue';
+import { createRouter, createWebHashHistory } from 'vue-router';
 import App from './App.vue';
 
 import './assets/main.css';
 
-createApp(App).mount('#app');
+//routes
+import GuidePageVue from './routes/guide/GuidePage.vue';
+import IndexPageVue from './routes/index/IndexPage.vue';
+import QuestionPageVue from './routes/question/QuestionPage.vue';
+import ResultPageVue from './routes/result/ResultPage.vue';
+import RecapPageVue from './routes/recap/RecapPage.vue';
+import ComparisonPageVue from './routes/comparison/ComparisonPage.vue';
+import DistrictSelectionPageVue from './routes/district-selection/DistrictSelectionPage.vue';
+
+export const appRoutes = {
+  index: {
+    name: 'index', //only for testing purposes
+    path: '/',
+    component: IndexPageVue,
+  },
+  districtSelection: {
+    name: 'district-selection',
+    path: '/:election',
+    alias: '/:election/district-selection',
+    component: DistrictSelectionPageVue,
+  },
+  guide: {
+    name: 'guide',
+    path: '/:election/:district/guide/:nr?',
```

Comment:
Pojďme zatím česky, pak se může nahodit něco [jako toto](https://bestofvue.com/repo/adbrosaci-vue-lang-router-vuejs-routing).

```suggestion
    path: '/:election/:district/navod/:nr?',
```